snowplow-tracker/src/androidTest/java/com/snowplowanalytics/snowplow/internal/tracker/PlatformContextTest.kt

```diff
@@ -246,6 +246,25 @@ class PlatformContextTest {
         Assert.assertEquals("12345678", sdjData[Parameters.MOBILE_LANGUAGE])
     }
 
+    @Test
+    fun catchesInvalidLocaleLanguageNoMocking() {
```

Comment:
I don't understand this test, if you are putting an invalid locale it´s dropping an exception, I thought the change was to return null if invalid? (catch (e: MissingResourceException) {
                null
            }) ?

Reply:
I wrote the test to replicate/find a bug, and didn't update it once I'd added the fix